src/dfx/src/commands/build.rs

```diff
@@ -90,5 +90,7 @@ pub fn exec(env: &dyn Environment, opts: CanisterBuildOpts) -> DfxResult {
         .with_env_file(env_file);
     runtime.block_on(canister_pool.build_or_fail(&env, logger, &build_config))?;
 
+    slog::info!(logger, "Finished building canisters.");
```

Comment:
```suggestion
    debug!(logger, "Finished building canisters.");
```
We just reduced the output of many commands drastically. The command exiting or the next step starting is IMO a good enough indication that building canisters has finished. I would skip this message unless we run with `--verbose`

Reply:
I didn't look closely enough and thought this would also show up in `dfx deploy`. I agree, having some finishing message make sense.

WDYT about getting rid of the `Building canisters...` message? If you run `dfx build` it's kind of redundant to say that as well. In `dfx deploy` it makes sense as a spinner since that's what's going on, but `dfx build` does nothing but building canisters

Reply:
Fixed in https://github.com/dfinity/sdk/pull/4123/commits/ea263bc916b4020e613fb3715834cc145530d3f1.